src/main/java/com/example/music/backend/user/dto/UserDto.java

```diff
@@ -21,11 +20,12 @@
     private Role role;
     private String dateOfBirth;
 
-    public UserDto(String userName, Gender gender, String email, String password, String dateOfBirch) {
+    public UserDto(String userName, Gender gender, String email, String password, String dateOfBirch, Long id) {
         this.username = userName;
         this.gender = gender;
         this.email = email;
         this.password = password;
         this.dateOfBirth = dateOfBirch;
+        this.id = id;
```

Comment:
Why do you need it inside DTO?

Reply:
I agree here id is not needed, I am doing PlaylistDto and id is needed there, confused